package.json

```diff
@@ -70,5 +72,8 @@
   "dependencies": {
     "classnames": "2.x",
     "fuzzysearch": "1.0.x"
-  }
+  },
+  "//TODO": [
```

Comment:
What do you want to do here?

Reply:
I found the only way to give a comment on json file is this way. And here is for "find another solution for icons from older react-icons version and remove this dependency." for the _@patternfly/react-icons": "3.15.17",_ I'll make it clearer in that TODO.